native/cocos/2d/renderer/RenderEntity.cpp

```diff
@@ -0,0 +1,11 @@
+#include <2d/renderer/RenderEntity.h>
+
+namespace cc {
+RenderEntity::RenderEntity(/* args */)
+{
+}
```

Comment:
If not any implementation, can just write like
```c++
RenderEntity::RenderEntity() = default;
```

Reply:
done